docs/changelog/139661.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 139661
+summary: Separate counts columns for multi-value binary doc value keywords
+area: Mapping
+type: enhancement
```

Comment:
Let's mark this as a non-issue for now. And this is now gated behind feature flag for new doc values options?